[callback-cross-realm-report-exception.html]
  [ResizeObserver reports the exception from its callback in the callback's global object]
    expected: FAIL
